patches/minecraft/net/minecraft/entity/item/EntityEnderPearl.java.patch

```diff
@@ -23,3 +23,19 @@
                  }
              }
              else if (entitylivingbase != null)
+@@ -141,13 +145,13 @@
+     }
+ 
+     @Nullable
+-    public Entity func_184204_a(int p_184204_1_)
```

Comment:
Where does this old method go? It hurts binary compatibility.

Reply:
It's still in `Entity`, which is where it's defined, so I think it's fine?

Reply:
Yes, this is good.